component/admin/layouts/payment/logs.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * @package     Redcore.Backend
+ * @subpackage  Layout
+ *
+ * @copyright   Copyright (C) 2012 - 2015 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later, see LICENSE.
+ */
+
+defined('_JEXEC') or die;
+
+$data = $displayData;
+
+$state = $data['state'];
+$items = $data['items'];
+$pagination = $data['pagination'];
+$formName = $data['formName'];
+$showToolbar = isset($data['showToolbar']) ? $data['showToolbar'] : false;
+$return = isset($data['return']) ? $data['return'] : null;
+$action = RRoute::_('index.php?option=com_redcore&view=payment_logs');
+
+// Allow to override the form action
+if (isset($data['action']))
+{
+	$action = $data['action'];
+}
+
+$paymentId = JFactory::getApplication()->input->getInt('id');
+$listOrder = $state->get('list.ordering');
+$listDirn = $state->get('list.direction');
+$saveOrder = $listOrder == 'ordering';
+
+$searchToolsOptions = array(
+	"searchFieldSelector" => "#filter_search_payment_logs",
+	"orderFieldSelector" => "#list_fullordering",
+	"searchField" => "search_payment_logs",
+	"limitFieldSelector" => "#list_payment_log_limit",
+	"activeOrder" => $listOrder,
+	"activeDirection" => $listDirn,
+	"formSelector" => ("#" . $formName),
+	"filtersHidden" => (bool) empty($data['activeFilters'])
+);
+?>
+<script type="text/javascript">
+	(function ($) {
+		$(document).ready(function () {
+			$('#<?php echo $formName; ?>').searchtools(
+				<?php echo json_encode($searchToolsOptions); ?>
+			);
+		});
+	})(jQuery);
+</script>
+	<?php
+	// Render the toolbar?
+	if ($showToolbar)
+	{
+		echo RLayoutHelper::render('payment.logs_toolbar', $data);
+	}
+	?>
+<form action="<?php echo $action; ?>" name="<?php echo $formName; ?>" class="adminForm" id="<?php echo $formName; ?>" method="post">
+	<?php
+	echo RLayoutHelper::render(
+		'searchtools.default',
+		array(
+			'view' => (object) array(
+					'filterForm' => $data['filterForm'],
+					'activeFilters' => $data['activeFilters']
+				),
+			'options' => $searchToolsOptions
+		)
+	);
+	?>
+
+	<hr/>
+	<?php if (empty($items)) : ?>
+		<div class="alert alert-info">
+			<button type="button" class="close" data-dismiss="alert">&times;</button>
+			<div class="pagination-centered">
+				<h3><?php echo JText::_('COM_REDCORE_NOTHING_TO_DISPLAY') ?></h3>
+			</div>
+		</div>
+	<?php else : ?>
+		<table class="table table-striped table-hover toggle-circle-filled" id="payment_logList">
+			<thead>
+			<tr>
+				<th width="1%">
+					<input type="checkbox" name="checkall-toggle" value=""
+					       title="<?php echo JText::_('JGLOBAL_CHECK_ALL'); ?>" onclick="Joomla.checkAll(this)"/>
+				</th>
+				<th width="20%" class="nowrap" data-hide="phone">
```

Comment:
I don't think you need these data-hide properties since footable is not in use in redCORE